config/default/manager_config_patch.yaml

```diff
@@ -13,7 +13,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: kubernetes-image-puller-operator
+  name: manager
```

Comment:
Because of the new `name-prefix` and resource renames, I plan to indicate in the KIP operator release notes what resource names have been changed